pandas/tests/window/test_rolling.py

```diff
@@ -1950,3 +1950,41 @@ def test_numeric_only_corr_cov_series(kernel, use_arg, numeric_only, dtype):
         op2 = getattr(rolling2, kernel)
         expected = op2(*arg2, numeric_only=numeric_only)
         tm.assert_series_equal(result, expected)
+
+
+@pytest.mark.parametrize("unit", ["s", "ms", "us", "ns"])
+@pytest.mark.parametrize("tz", [None, "UTC"])
+def test_rolling_rolling_sum_window_microseconds_conflict_timestamp(unit, tz):
```

Comment:
```suggestion
def test_rolling_timedelta_window_non_nanoseconds(unit, tz):
```

Let's simplify the test name a bit here (it's not specific about microseconds or sum; but the test is now generalized to test all resolutions, and it is specific about using a timedelta window (not fixed number of rows))